packages/react-conformance/src/utils/merge.tsx

```diff
@@ -0,0 +1,41 @@
+import * as _ from 'lodash';
```

Comment:
You shouldn't need to use lodash for merging here. Probably best to just copy the implementation from scripts/tasks/merge.js unless that didn't work for some reason.

Reply:
I'm only using lodash to check for types, if a property exists in the object and to insert an attribute that doesn't exist in the first object. I'm basing everything in obj1 and copying everything else from obj2.

Reply:
You don't need lodash for either of those things. For `has` you can do `!!obj1 && obj1.hasOwnProperty(k)`. For `isArray` you can do `Array.isArray(foo)`.

Reply:
I see, will change it!